src/app/public/modules/phone-field/phone-field.component.html

```diff
@@ -1,45 +1,96 @@
 <div
   class="sky-form-group sky-input-group sky-phone-field"
+  [@initalLoad]
 >
   <div
     class="sky-input-group-btn sky-phone-field-country-btn"
   >
-    <sky-dropdown
+    <button
+      aria-live="polite"
+      class="sky-btn sky-btn-default"
+      [attr.title]="('skyux_phone_field_country_select_label' | skyAppResources) + ' ' + ('skyux_phone_field_country_selected_label' | skyAppResources:selectedCountry.name)"
+      [attr.aria-label]="('skyux_phone_field_country_select_label' | skyAppResources) + ' ' + ('skyux_phone_field_country_selected_label' | skyAppResources:selectedCountry.name)"
       [disabled]="countrySelectDisabled"
-      [label]="'skyux_phone_field_dropdown_label' | skyLibResources"
-      [title]="selectedCountry.name + ' +' + selectedCountry.dialCode"
+      (click)="toggleCountrySearch(true)"
     >
-      <sky-dropdown-button>
-        <div
-          [className]="'iti-flag ' + selectedCountry.iso2"
+      <div
+        [className]="'iti-flag ' + selectedCountry.iso2"
+      >
+      </div>
+    </button>
+  </div>
+  <ng-container *ngIf="phoneInputShown">
+    <span
+      class="sky-phone-field-container"
+      [@phoneInputAnimation]
+      (@phoneInputAnimation.done)="phoneInputAnimationEnd($event)"
+    >
+      <ng-content></ng-content>
+    </span>
+  </ng-container>
+  <ng-container *ngIf="countrySearchShown">
+    <span
+      class="sky-phone-field-container"
+      [@countrySearchAnimation]
+      (@countrySearchAnimation.done)="countrySearchAnimationEnd($event)"
+
+    >
+      <form
+        novalidate
+        class="sky-phone-field-country-search"
+        [formGroup]="countrySearchForm"
+      >
+        <sky-autocomplete
+          [data]="countries"
+          [propertiesToSearch]="['name', 'iso2']"
+          [searchResultTemplate]="countrySearchResultTemplate"
+          [searchTextMinimumCharacters]="2"
+          (selectionChange)="onCountrySelected($event)"
         >
-        </div>
-      </sky-dropdown-button>
-      <sky-dropdown-menu>
-        <sky-dropdown-item *ngFor="let country of countries; let i = index"
-          [ngClass]="{
-            'sky-phone-field-country-selected': i === 0,
-            'sky-phone-field-country-default': country.iso2 === defaultCountry
-          }"
+          <input
+            skyAutocomplete
+            [attr.aria-label]="'skyux_phone_field_country_search_label' | skyAppResources"
```

Comment:
Input properties should go after normal attributes.

Reply:
Good catch. That one got converted but not moved. Done